src/overseer/heartbeat.clj

```diff
@@ -87,6 +87,14 @@
           (timbre/warn (str "Resetting: " (string/join ", " (map :job/id jobs))))
           @(d/transact conn dead-job-txns))
         (Thread/sleep (+ (config/heartbeat-sleep-time config) (sleep-stagger))))
+      (catch java.util.concurrent.ExecutionException ex
+        (if (= :db.error/cas-failed
+               (->> ex
+                    Throwable->map
+                    :data
+                    :db/error))
+          (timbre/info "Heartbeat monitor CAS failed; ignoring.")
```

Comment:
If this behavior is expected all the time (literally every time a heartbeat reset needs to occur, 30 boxes are going to race and 29 of them are going to have this "failure"), then I don't think it's warn-worthy?